### PR TITLE
Eg mod

### PR DESCRIPTION
**Pull request by swaythe**
Removed the example file nipype_preproc_spm_nyu because 1) it errors, and 2) it uses the NYU rest fetcher which is deprecated. I will rework this example with a different dataset later because it showcases the use of an ini file.

The example coreg_demos uses the ABIDE dataset with hardcoded paths; removed it.

Added a line to the README to say that pypreprocess works with Ubuntu 16 and 18, no guarantees otherwise.

### CODE REVIEW

File: examples/pipelining/nyu_rest_preproc.ini [hunk 1]
```diff
@@ -30,12 +30,12 @@
 
 # Path (relative or full) of directory containing data (if different from directory
 # containing this configuration file).
-dataset_dir = 
+dataset_dir =
```
Comment (bthirion) [inline examples/pipelining/nyu_rest_preproc.ini:33]: Something wrong here

**Review by bthirion — COMMENTED** [review]
Almost there, thx !

File: examples/pipelining/nyu_rest_preproc.ini (hunk 1 above)
Comment (swaythe) [reply examples/pipelining/nyu_rest_preproc.ini]: I've removed the NYU rest ini file. The NYU rest example is obviously not the only one showcasing use of ini files. Was there a specific reason for including this dataset?

Comment (bthirion) [reply examples/pipelining/nyu_rest_preproc.ini]: O that's an ini file, sorry.
Anyway, if the fetcher is deprecated, it's better to remove it (as you may have seen, Nilearn has examples on a new resting state dataset).
Best,
B

**Review by bthirion — APPROVED** [review]

**Comment by bthirion** [comment]
Maybe a rebase to remove conflicts ?
Best,
B

**Comment by swaythe** [comment]
@bthirion The conflicts have been resolved, should I merge once the tests are complete?

**Pull request merged**